paper/paper.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'OpenTURNS: A Python package for uncertainty quantification'
+tags:
+  - Python
+  - uncertainty quantification
+  - probabilistic modeling
+  - statistics
+  - meta-modeling
+  - calibration
+  - stochastic processes
+  - Bayesian modeling
+  - machine learning
+  - supervised machine learning
+  - unsupervised machine learning
+  - surrogate
+authors:
+  - name: Michaël Baudin
+    orcid: 0000-0002-0450-7947
+    affiliation: 1
+  - name: Anne Dutfoy
+    affiliation: 1
+  - name: Joseph Muré
+    affiliation: 1
+  - name: Julien Pelamatti
+    affiliation: 1
+  - name: Régis Lebrun
+    affiliation: 2
+  - name: Sofiane Haddad
+    affiliation: 2
+  - name: Julien Schueller
+    affiliation: 3
+  - name: Loïc Brevault
+    affiliation: 4
+  - name: Mathieu Balesdent
+    affiliation: 4
+affiliations:
+ - name: EDF R&D, France
+   index: 1
+ - name: Airbus, France
+   index: 2
+ - name: Phimeca, France
+   index: 3
+ - name: ONERA, France
+   index: 4
+date: 30 April 2025
+bibliography: paper.bib
+---
+
+# Summary
+
+``OpenTURNS`` is a C++ library for uncertainty propagation by probabilistic methods.
+OpenTURNS is also available as a Python module. 
+
+## Introduction
+Numerical simulation and the uncertainties associated with these models make decision-making robust in the design and operation of complex systems. Supervisory authorities in safety-critical industrial sectors (e.g. nuclear power or civil aviation) are demanding that players use tried-and-tested methods, backed up by tested, documented and auditable implementations. As a result, several uncertainty processing libraries have emerged over the last few decades. However, many of these approaches are dedicated to a particular discipline, such as numerical fluid mechanics or hydrology. This approach limits the sharing of best practices. This is why a generic approach to uncertainty management is so important.
```

Comment:
Some minor suggestions : 
-  "Numerical simulation and the uncertainties associated with these models make decision-making robust in the design and operation of complex systems." --> Numerical simulation and the uncertainties inherent in these models play a crucial role in supporting robust decision-making for the design and operation of complex systems.
- "are demanding that players use tried-and-tested methods" --> require the use of proven methods supported by well-tested, documented, and auditable implementations. 
- "This is why a generic approach to uncertainty management is so important." --> This highlights the need for a generic, cross-disciplinary approach to uncertainty management.